pkg/manifest/simple/manifest_test.go

```diff
@@ -5,23 +5,19 @@
 package simple_test
 
 import (
-	"crypto/rand"
-	"encoding/hex"
 	"errors"
 	"reflect"
 	"testing"
 
 	"github.com/ethersphere/bee/pkg/manifest/simple"
+	"github.com/ethersphere/bee/pkg/swarm"
 )
 
 // randomAddress generates a random address.
-func randomAddress() string {
-	b := make([]byte, 32)
-	_, err := rand.Read(b)
-	if err != nil {
-		panic(err)
-	}
-	return hex.EncodeToString(b)
+func randomAddress(t *testing.T) string {
```

Comment:
This seems function seems to be unnecessary since it can be replaced with `swarm.RandAddress(t).String()`.